conf/pacific/upgrades/5-0_upgrades.yaml

```diff
@@ -0,0 +1,43 @@
+globals:
+  - ceph-cluster:
+     name: ceph
+     node1:
+       role:
+         - _admin
+         - mon
+         - mgr
+         - installer
+     node2:
+       role:
+         - mon
+         - mgr
+     node3:
+       role:
+         - mon
+         - mgr
+     node4:
+       role:
+         - osd
+         - mds
+       no-of-volumes: 3
+       disk-size: 20
+     node5:
+       role:
+         - osd
+         - rgw
+         - iscsi-gw
+       no-of-volumes: 3
+       disk-size: 20
+     node6:
+       role:
+         - osd
+         - rgw
+       no-of-volumes: 3
+       disk-size: 20
+     node7:
+       role: iscsi-gw
+     node8:
+       role: client
+     node9:
```

Comment:
Can we colocate `grafana` with `iscsi-gw`?

Reply:
Yes!
Provided that we add osd to that node, which we had to do in /upgrades.yaml
Ref - https://access.redhat.com/articles/1548993
Shall I add an OSD and merge grafana node in both suites ?

Reply:
Yes, please. If it suits your requirements let's go ahead with the change.